src/ui/components/home/site_summary.rs

```diff
@@ -83,51 +84,51 @@ pub fn SiteSummary() -> impl IntoView {
           </Unpack>
           <section aria-labelledby="instance-stats-heading" class="my-4">
             <h3 id="instance-stats-heading" class="text-2xl font-bold mb-2">
-              Instance Stats
+              {tr!("instance-stats")}
             </h3>
             <Unpack item=counts let:counts>
               <div class="font-semibold flex flex-wrap *:m-1.5">
                 <div>
                   <Icon icon=IconType::Posts size=IconSize::Large class="inline" />
                   {counts.posts.si_format().to_string()}
                   " "
-                  <span class="text-sm">Posts</span>
+                  <span class="text-sm">{tr!("posts")}</span>
                 </div>
                 <div>
                   <Icon icon=IconType::Comments size=IconSize::Large class="inline" />
                   {counts.comments.si_format().to_string()}
                   " "
-                  <span class="text-sm">Comments</span>
+                  <span class="text-sm">{tr!("comments")}</span>
                 </div>
               </div>
               <table class="w-full mt-3 table shadow-lg">
                 <caption class="text-lg font-semibold whitespace-nowrap align-middle text-start mb-2">
                   <Icon icon=IconType::Users size=IconSize::Large class="inline me-2" />
-                  Active Users
+                  {tr!("active-users")}
                 </caption>
                 <thead>
                   <tr class="font-extrabold text-sm bg-base-300 *:p-3">
                     <th class="text-start" scope="col">
-                      Time Frame
+                      {tr!("time-frame")}
                     </th>
                     <th class="text-center" scope="col">
-                      Count
+                      {tr!("count")}
                     </th>
                   </tr>
                 </thead>
                 <tbody class="bg-base-100">
-                  <UserStatRow text="Today" count=counts.users_active_day />
-                  <UserStatRow text="Past Week" count=counts.users_active_week />
-                  <UserStatRow text="Past Month" count=counts.users_active_month />
-                  <UserStatRow text="Past 6 Months" count=counts.users_active_month />
-                  <UserStatRow text="All Time" count=counts.users_active_month />
+                  <UserStatRow text={tr!("today")} count=counts.users_active_day />
```

Comment:
These wouldn't work without the wrapping braces. I have no clue why as we pass the result of the `tr!` macro to attributes without the braces in several other places in the app. Annoyingly, leptos format strips these braces, causing the project to fail to compile. I'm working on reproducing the bug in a sample project and will open an issue once I manage to reproduce it.